### PR TITLE
feat: allow user to select an image to render in SOF.

### PR DESCRIPTION
**Pull request by benoitdevos**
logion-network/logion-internal#468

### CODE REVIEW

File: src/loc/statement/StatementOfFactsForm.tsx [hunk 1]
```diff
@@ -8,6 +8,8 @@ import { useCommonContext } from "../../common/CommonContext";
 import { FormValues, Language } from "./PathModel";
 
 import "./StatementOfFactsForm.css"
+import FileSelectorButton from "../../common/FileSelectorButton";
+import React from "react";
```
Comment (gdethier) [inline src/loc/statement/StatementOfFactsForm.tsx:12]: Not sure this import is still needed.

File: src/loc/statement/StatementOfFactsTemplateEN.tsx [hunk 2]
```diff
@@ -27,6 +27,12 @@ export default function StatementOfFactsTemplateEN(props: Props) {
                 <div>{ props.pathModel.polkadotAddress }</div>
             </div>
 
+            { props.pathModel.imageSrc.startsWith("data:image") &&
```
Comment (gdethier) [inline src/loc/statement/StatementOfFactsTemplateEN.tsx:30]: Why this condition? We might as well pass a regular URL and it should work the same, no?

File: src/loc/statement/StatementOfFactsTemplateFR.tsx [hunk 3]
```diff
@@ -27,6 +27,12 @@ export default function StatementOfFactsTemplateFR(props: Props) {
                 <div>{ props.pathModel.polkadotAddress }</div>
             </div>
 
+            { props.pathModel.imageSrc.startsWith("data:image") &&
```
Comment (gdethier) [inline src/loc/statement/StatementOfFactsTemplateFR.tsx:30]: Same question as above.

**Review by gdethier — APPROVED** [review]
Good job! Just a couple of questions.

File: src/loc/statement/StatementOfFactsForm.tsx (hunk 1 above)
Comment (benoitdevos) [reply src/loc/statement/StatementOfFactsForm.tsx]: Correct. https://github.com/logion-network/logion-wallet/compare/1aa682cc039e90c541a64b542695a4a2368bc21d..89d5ef7b968f05b934e1e5c45ae8bc6f466bc6a3

File: src/loc/statement/StatementOfFactsTemplateEN.tsx (hunk 2 above)
Comment (benoitdevos) [reply src/loc/statement/StatementOfFactsTemplateEN.tsx]: Adapted: https://github.com/logion-network/logion-wallet/compare/1aa682cc039e90c541a64b542695a4a2368bc21d..89d5ef7b968f05b934e1e5c45ae8bc6f466bc6a3

**Pull request merged**